<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFMediaPlayer_64Class</class>
 <widget class="QWidget" name="FFMediaPlayer_64Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFMediaPlayer_64</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="Show" native="true">
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>95</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="13">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Play">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="TotalTime">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Time">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="Curr">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="high">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>720P</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="low">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>480P</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="SavePic">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>截图</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="Speed1">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="Speed1_25">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>1.25</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="Total_2">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../视频播放器/Media-Build/voice (2).png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="maximumSize">
         <size>
          <width>269</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>392</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="12">
       <widget class="QPushButton" name="AddMedia">
        <property name="text">
         <string>添加视频</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FFMediaPlayer_64.qrc"/>
 </resources>
 <connections/>
</ui>
